src/stories/jobs/progressBar.stories.mdx

```diff
@@ -13,12 +13,27 @@ The **ProgressBar** component is an
 used to visually communciate the completion of `Steps` within a `Job`
 installation.
 
-<Story id="jobs-progressbar-example--progress-bar-component" />
+<Story
+  id="jobs-progressbar-example--progress-bar-component&args=job:Complete"
+  height="auto"
+/>
 
 The **ProgressBar** appears once an installation has started. The component is
-labeled "Installation Progress:" and updates the percentage of `Steps` completed
-as the install continues, represented by a number and a colored bar.
+labeled "Installation Progress:" and updates the percentage of `Steps`
+completed. The completed `Steps` percentage is represented by a colored bar and
+a number that grow as the install continues.
+
+<Story
+  id="jobs-progressbar-example--progress-bar-component&args=job:Started"
```

Comment:
@jgerigmeyer For some reason the various states show on the component example page but not here? Does this have something to do with how the original component is made? Or am I missing something in how I'm rendering the component in storybook?